### PR TITLE
feat: solve PQIRs

### LINKED ISSUE #165: Marking PQI as Solved
**User input 1**
Possibility to mark problems, ideas and questions as solved. Now it's only possible to delete them (not desirable as it is part of the meta-data needed).

**User input 2**
When a problem is deleted from a flow, it still is registered as an "existing problem" and still appears in "categorize PQIR's"

**Sharepoint ID(s)**
49, 105

**Requirements (for @ntorsen)**
- [x] Checkbox next to problem or risk with hover text "Solve" or "Mitigate", respectively.
- [x] Problem or risk disappear from canvas when marked as solved.
- [x] Problem or risk are placed under section "Solved or mitigated" in the card side sheet, and are grayed out.
- [x] Description of solved problem or risk is visible.
- [x] Checkbox next to solved problem or riks with hover text "Unsolve" or "Unmitigate"? (@Linedueo can you confirm the use of these two words?)
- [x] Solved PQIR's should not show in the categorisation page.

### PR DESCRIPTION
**Pull request by SjoenH**
Possibility to mark problems, ideas and questions as solved.

fix #165

### CODE REVIEW

File: components/EditTaskSection.tsx [hunk 1]
```diff
@@ -26,6 +34,36 @@ export function EditTaskSection(props: {
...
+  // const patchTaskMutation = useMutation(
+  //   (patchedTask: taskObject) => patchTask(patchedTask),
+  //   {
+  //     onSuccess() {
+  //       // notifyOthers("Deleted a task", id, account);
+  //       return queryClient.invalidateQueries();
+  //     },
+  //     onError: (e) => dispatch.setSnackMessage(unknownErrorToString(e)),
+  //   }
+  // );
```
Comment (dansvend) [inline components/EditTaskSection.tsx:47]: Consider removing this

File: components/EditTaskSection.tsx [hunk 2]
```diff
@@ -26,6 +34,36 @@ export function EditTaskSection(props: {
...
+      solvedTask,
+      solved,
+    }: {
+      card: vsmObject;
+      solvedTask: taskObject;
+      solved: boolean;
+    }) => solveTask(card.vsmObjectID, solvedTask.vsmTaskID, solved),
+    {
+      onSuccess() {
+        // todo notify others
```
Comment (dansvend) [inline components/EditTaskSection.tsx:60]: Should this be done now?

**Review by dansvend — APPROVED** [review]
Clean code👌 Just a few considerations

File: components/EditTaskSection.tsx (hunk 1 above)
Comment (SjoenH) [reply components/EditTaskSection.tsx:47]: Removed

**Pull request merged**